### DISCUSSION TITLE
How to create a smooth animation with particles

### DISCUSSION THREAD

**Discussion by Drosaca** [tsparticles/react#113] [Q&A] [answered]
I would like to decrease the speed of particles until complete stop but when i tried it by using a state on the input options the result i got looks more to a shaking effect than slowing down, is there a conventional way of doing it ? 
thanks

**Comment by matteobruni** [comment, marked as answer]
Use `particles.move.decay` option, sample here: https://codepen.io/matteobruni/pen/RwmMbWq
